<!DOCTYPE html>
<html><head><title>ErrorResponseException</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">1 instantiation of ErrorResponseException</div><div class="rA">Microsoft.Azure.Management.ResourceGraph (1)</div><div class="rG" id="Microsoft.Azure.Management.ResourceGraph"><div class="rF"><div class="rN">Generated\ResourceGraphClient.cs (1)</div>
<a href="../Generated/ResourceGraphClient.cs.html#484"><b>484</b>var ex = new <i>ErrorResponseException</i>(string.Format(&quot;Operation returned an invalid status code &#39;{0}&#39;&quot;, _statusCode));</a>
</div>
</div>
<div class="rH">3 references to ErrorResponseException</div><div class="rA">Microsoft.Azure.Management.ResourceGraph (2)</div><div class="rG" id="Microsoft.Azure.Management.ResourceGraph"><div class="rF"><div class="rN">Generated\ResourceGraphClient.cs (2)</div>
<a href="../Generated/ResourceGraphClient.cs.html#370"><b>370</b>/// &lt;exception cref=&quot;<i>ErrorResponseException</i>&quot;&gt;</a>
<a href="../Generated/ResourceGraphClient.cs.html#484"><b>484</b><i>var</i> ex = new ErrorResponseException(string.Format(&quot;Operation returned an invalid status code &#39;{0}&#39;&quot;, _statusCode));</a>
</div>
</div>
<div class="rA">Microsoft.Azure.Management.ResourceGraph.Tests (1)</div><div class="rG" id="Microsoft.Azure.Management.ResourceGraph.Tests"><div class="rF"><div class="rN">ResourceGraphTests.cs (1)</div>
<a href="../../Microsoft.Azure.Management.ResourceGraph.Tests/ResourceGraphTests.cs.html#239"><b>239</b>Assert.Throws&lt;<i>ErrorResponseException</i>&gt;(</a>
</div>
</div>
</body></html>